### ISSUE TITLE
Bug: Repairing file or HTTP 503 

### ISSUE THREAD

**Issue by herbhuang** [debridmediamanager/zurg-testing#54]
**Expected Behavior**
Correctly server `http` or `webdav` files.

**Actual Behavior**
Time out on http download. `HTTP error 25500269` on Infuse. In logs, it keeps repairing files. Through DMM, it produced a lot of duplicates. 

**Detailed Steps**
Follow instruction, running through `docker compose`. Initial set up, have to do the same to #40 to prevent `zurg` from exiting. After that, it stays 
```2024-01-25T05:02:15.566Z	INFO	zurg	Starting server on [::]:9999
2024-01-25T05:02:15.566Z	INFO	manager	Starting periodic refresh
2024-01-25T05:02:16.005Z	DEBUG	manager	Found 0 torrents that are no longer cached
2024-01-25T05:02:16.005Z	DEBUG	manager	Found 0 broken torrents to repair in total
```
until I started to query file via http or webdav.

**Operating System**
```
 OS                 : Debian GNU/Linux 12
 Arch               : aarch64 (64 Bit)
 Kernel             : 6.1.0-17-cloud-arm64
```

**Environment Setup**
```
Server:
 Engine:
  Version:          20.10.24+dfsg1
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.19.8
  Git commit:       5d6db84
  Built:            Thu May 18 08:41:26 2023
  OS/Arch:          linux/arm64
  Experimental:     true

zurg
Built At: 2024-01-25T00:18:52
Commit: b2bd188736217fbe98c7aa275714193b89145723
Version: v0.9.3-hotfix.2
```
```
version: "3.8"
services:
  zurg:
    image: ghcr.io/debridmediamanager/zurg-testing:latest
    restart: unless-stopped
    ports:
      - 32003:9999
    volumes:
      - ./plex_update.sh:/app/plex_update.sh
      - ./config.yml:/app/config.yml
      - zurgdata:/app/data
  rclone:
    image: rclone/rclone:latest
    restart: unless-stopped
    environment:
      TZ: America/Chicago
      PUID: 1000
      PGID: 1000
    volumes:
      - /media/zurg:/data:rshared
      - ./rclone.conf:/config/rclone/rclone.conf
    cap_add:
      - SYS_ADMIN
    security_opt:
      - apparmor:unconfined
    devices:
      - /dev/fuse:/dev/fuse:rwm
    depends_on:
      - zurg
    command: "mount zurg: /data --allow-non-empty --allow-other --dir-cache-time 10s --vfs-cache-mode
      full"
volumes:
  zurgdata: 
```

**Logs**
Provide a link to [Zurg logs](https://zurg.herbo.cc/logs/). 

**Rclone Configuration**
```
[zurg]
type = webdav
url = http://zurg:9999/dav
vendor = other
pacer_min_sleep = 0

[zurghttp]
type = http
url = http://zurg:9999/http
no_head = false
no_slash = false
```

**Zurg Configuration**
```
host: "[::]"
port: 9999
concurrent_workers: 20
check_for_changes_every_secs: 15
ignore_renames: false
retain_rd_torrent_name: false
retain_folder_name_extension: false
auto_delete_rar_torrents: false
use_download_cache: true
enable_repair: true
network_buffer_size: 10485760 # 1 MiB
serve_from_rclone: false
verify_download_link: true

# Run ./zurg network-test
preferred_hosts:
  - 67.download.real-debrid.cloud # hops: 7 latency: 1.475ms
  - 63.download.real-debrid.cloud # hops: 7 latency: 2.256ms

```

**Screenshots**

**Comment by kudddyyy** [comment]
![image](https://github.com/debridmediamanager/zurg-testing/assets/155233331/e1a5b5fc-3d16-4ad0-8980-9da7fb83f4e9)
Looks like this may be a copyright issue

**Comment by herbhuang** [comment]
> Hi what are those files that are in progress? Also are you able to share me the torrent "REMUX.1080p.Insidious...."

I use plex_debrid to add files into debrid so I am not sure what they are.
Also because of these, I don't have the torrent files. Do mag links work? I have listed three.
```
[insidious] magnet:?xt=urn:btih:fbd4cbee5aba3253e2b4526ccd31166249258158
[inu-oh] magnet:?xt=urn:btih:bc64554fa8b5a7af7493b78cf8e4a73ca03af062
[close] magnet:?xt=urn:btih:d0be3e099d2c3377090274b303036c182a385469
```

**Comment by herbhuang** [comment]
> ![image](https://private-user-images.githubusercontent.com/155233331/299580143-e1a5b5fc-3d16-4ad0-8980-9da7fb83f4e9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxOTY2MjgsIm5iZiI6MTcwNjE5NjMyOCwicGF0aCI6Ii8xNTUyMzMzMzEvMjk5NTgwMTQzLWUxYTViNWZjLTNkMTYtNGFkMC04OTgwLTlkYTdmYjgzZjRlOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyNVQxNTI1MjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZTAxZDRjNDYyYTk5MjIzOWQ0YmRhNTIwMjI3NDhlYWMwNWI2OWIwNjcwYmJhOGEyMTI4YWI3N2JkNzM1MWJhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.zx9TKlhaR65BcMR-GmU1WgFukcHbY9aCHfFIJm2V0_M) Looks like this may be a copyright issue

Thanks for another check. I did not realize that could be a problem. I checked RD and found I can still download and stream it ok. Also, WebDav from RD also works for me.

**Comment by Varming73** [comment]
I also see a large amount of repairs of the same files over and over.

**Comment by yowmamasita** [comment]
Fixed on the latest version `v0.9.3-hotfix.3`
